### DISCUSSION TITLE
Feature Request: Store object file in a key-value store

### DISCUSSION THREAD

**Discussion by junghans** [ccache/ccache#281] [Ideas]
How hard would it be to store the cached object files in an in-memory key-value store, e.g redis, which then can be shared between different machines. I think that could be very useful for compile farms.

**Comment by afbjorklund** [comment]
See the `dev/memcached` branch, and #58

Not only objects (etc), but also manifests ?

**Comment by afbjorklund** [comment]
At one point we looked at supporting multiple databases (and multiple compressions, too)... But that just seems to have lead to people sticking even harder to nfs files (and gzip -1) ?

So now I am trying to keep to memcached (and lz4f) as being the "best" alternative  options... I still believe that to be true, but it is mostly to keep the complexity down.
